Readme.md

```diff
@@ -34,11 +34,21 @@ functions:
             - ${bref:layer.console}
 ```
 
+### Extra configuration files
+
+You may add extra configuration files for PHP in the ``php/conf.d/`` directory.
+
 ```ini
-;php/conf.d/php.ini
+;php/conf.d/amqp.ini
+; Loading the AMQP extension.
 extension=/opt/bref-extra/amqp.so
 ```
 
+```ini
+;php/conf.d/php.ini
+memory_limit=128M
```

Comment:
This is not really needed, right?

Reply:
It shows that one can also modify any PHP setting